src/styles/stars.css

```diff
@@ -0,0 +1,23 @@
+body {
+  margin: 0;
+  padding: 0;
+  border: 0;
+  font-family: Arial, Helvetica, sans-serif;
+  box-sizing: border-box;
```

Comment:
It's recommended not to use '*' selector as it impacts performance. It would be better to set styles only for elements that require them.